coverage/clover.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These files shouldn't be committed to repo

Reply:
forgot to add to .gitignore